Singularity/Singularity/Sound/SoundManager.cs

```diff
@@ -0,0 +1,482 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mime;
+using System.Security.Policy;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Audio;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Media;
+
+namespace Singularity.Sound
+{
+    public class SoundManager
+    {
+
+        // Sound listener for 3D audio.
+        private readonly AudioListener mListener;
+        // The z coordinate for 3D audio.
+        private float mSoundPlaneDepth;
+        // Dictionary containing all the songs assigned to their names and sorted by phase of the game.
```

Comment:
Dictionaries don't have a sort so what do you mean by sorted by phase?

Reply:
True, what I meant was "paired with". I changed it.